lib/plugins/acts_as_commentable/commentable_entity.rb

```diff
@@ -27,7 +27,7 @@ def count_all_comments
   # hidden-by-admin comments.
   # returns number of visible (not deleted) comments
   def count_visible_comments
-    self.total_comments.count(:all, :conditions => {:hidden_by_admin => false, :is_deleted => false})
+    self.total_comments.count(:all, :conditions => {:hidden_by_admin => false, :is_deleted => false, :unreviewed => false})
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Redundant `self` detected.<br>Space inside { missing.<br>Space inside } missing.